kube-derive/tests/resource.rs

```diff
@@ -0,0 +1,55 @@
+use k8s_openapi::{
+    api::core::v1::{ConfigMap, Secret},
+    ByteString,
+};
+use kube::api::ObjectMeta;
+use kube_derive::Resource;
+
+#[derive(Resource, Default)]
+#[inherit(resource = "ConfigMap")]
```

Comment:
```suggestion
#[derive(Resource, Default)]
#[resource(inherit = "ConfigMap")]
```
maybe we can even flip these so that the resource is inherited (opening up in the future for alternate methods)
(the key here could be named something else, but feel like the outer one should match the macro)